pkg/apis/networking/register.go

```diff
@@ -37,4 +37,8 @@ const (
 	// IngressLabelKey is the label key attached to underlying network programming
 	// resources to indicate which ClusterIngress triggered their creation.
 	IngressLabelKey = GroupName + "/clusteringress"
+
+	// IngressClassAnnotationKey value for specifying knative's Istio
+	// ClusterIngress reconciler.
+	IstioIngressClassName = "istio.ingress.networking.knative.dev"
```

Comment:
Better to move this into the ingress controller file?
I consider this self maintained by related controller, and we should not be aware of specific keys here in in-tree API.

Reply:
Hmm, we were going with flat names in the autoscaler, but I do think that this convention is better (and closer to K8s Ingress).  +1 on moving it next to the implementation.